man/bundle-doctor.ronn

```diff
@@ -0,0 +1,33 @@
+bundle-doctor(1) -- Checks the bundle for common problems
+=========================================================
+
+## SYNOPSIS
+
+`bundle doctor` [--quiet]
+                [--gemfile=GEMFILE]
+
+## DESCRIPTION
+
+Checks your Gemfile and gem environment for common problems. If issues
+are detected, Bundler prints them and exists status 1. Otherwise,
+Bundler prints a success message and exists status 0.
```

Comment:
(Awesome that you decided to build a man page for this!)

Typo: `exists` => `exits`.

Suggestion: `exists status 1` => `exits with status 1`. Same, but for the `status 0` part of this paragraph.

Reply:
Thanks for the good catch! I fixed the typos in https://github.com/bundler/bundler/pull/6271/commits/c5f0852bf9eb585ebc36fbeedc4627d6492f2731.